.github/actions/install-openvino/install.sh

```diff
@@ -2,15 +2,23 @@
 
 set -e
 
+if [ "$#" -ne 1 ]; then
+    version="2020.4.287"
+else
+    version="$1"
+fi
+
+scriptdir="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+
```

Comment:
👍 This is a helpful change for users who may want to reuse this script to install OpenVINO; not sure why the paths seem messed up in the CI (I usually use a simpler version of "find the directory this script lives in," FWIW). If you want, you could use Bash variables defaults for the version, `${1:-"..."}`.

Reply:
I don't know bash very well, this is just a copypaste of what stackoverflow says to do... I'll see what went wrong in CI

Reply:
The script error was a simple typo, should be fixed now